lambda: 90
abdRatio: 2
minSegPts: 10
fitMethod: 0
laserScanTopic: /laser_horizontal_front

LM:
    lambdaIni: 0.001

Q: [1, 0,  0,  0,
    0, 1,  0,  0,
    0, 0, 10,  0,
    0, 0,  0, 10]

R: [10, 0,
    0, 10]

P0: [0.1,   0,   0,   0,
       0, 0.1,   0,   0,
       0,   0, 1,   0,
       0,   0,   0, 1]